src/components/weather/DailyWeather.tsx

```diff
@@ -35,7 +36,10 @@ export const DailyWeather = ({ date, icon, temperatures }: Props) => {
           <WrapperRow center>
             <View style={styles.iconContainer}>
               <Image
-                source={{ uri: `https://openweathermap.org/img/wn/${icon}@2x.png` }}
+                source={{
+                  uri: `https://openweathermap.org/img/wn/${icon}@2x.png`,
+                  captionText: desctription
```

Comment:
when VoiceOver was saying "grafik" before, what is the result now for some case?

Reply:
As you can see from the sample code I have shared below, it first reads the description and then reads the (Bild) text that was previously added to the code. 

For example: `Überwiegend bewölkt Bild`

```
accessibilityLabel={`${sourceProp.captionText ? sourceProp.captionText : ''} ${consts.a11yLabel.image}`}
```
`consts.a11yLabel.image = (Bild)`

Reply:
i understand! very good.